src/app/components/pages/requests/forms/request/request.component.html

```diff
@@ -240,7 +240,7 @@
         <mat-label><span>{{ 'REQUEST.TIME_ABROAD' | translate }}</span></mat-label>
         <span matPrefix class="asterisk" *ngIf="!selfInsuredCheckbox.checked && useCase === 1">*</span>
         <mat-date-range-input [min]="tomorrow" [rangePicker]="insurancePicker" [disabled]="useCase > 1 || selfInsuredCheckbox.checked" #abroadDateInsurance (click)="insurancePicker.open()">
-          <input #abroadDateInsuranceStart matStartDate placeholder="{{ 'REQUEST.TIME_ABROAD_FROM' | translate }}" readonly>
+          <input #abroadDateInsuranceStart matStartDate placeholder="{{ 'REQUEST.TIME_ABROAD_FROM' | translate }}" readonly (dateChange)="saveFormDataToLocalStorage()">
           <input #abroadDateInsuranceEnd matEndDate placeholder="{{ 'REQUEST.TIME_ABROAD_TO' | translate }}" readonly>
```

Comment:
there is no (dateChange) method on EndDate?

Reply:
when changing EndDate, event dateChange on StartDate was firing too xd. I don't know if this is bug or what